code/modules/antagonists/blob/blob_mobs.dm

```diff
@@ -103,15 +100,15 @@
 	verb_yell = "psychically screams"
 	melee_damage_lower = 2
 	melee_damage_upper = 4
-	obj_damage = 20
-	environment_smash = ENVIRONMENT_SMASH_STRUCTURES
+	environment_smash = ENVIRONMENT_SMASH_NONE
+	obj_damage = 0
```

Comment:
by the way, I wouldn't be opposed to making zombie-less blob spores have a base obj_damage of 5 instead of 0; let me know what you think in the comments of this PR (or with emotes).